test/hotspot/jtreg/compiler/loopstripmining/LoadSplitThruPhi.java

```diff
@@ -54,6 +67,23 @@ public static void main(String[] args) {
         for (int i = 0; i < 1000000; i++) {
             getPermutations(inputArray, outputArray);
         }
+
+        for (int i = 0; i < 100; ++i) {
+            Thread t = new Thread(new Runnable() {
```

Comment:
Why do you need a new thread?

Reply:
`for (int a = 0; a < (int)(a + 16); a++) {` only will terminate when `a` overflows, so I add a thread to execute the test,  and use main thread to control the test time. I will refactor the test if the final fix is done.